### DISCUSSION TITLE
Loading Elementor page via PHP

### DISCUSSION THREAD

**Discussion by Ferdinand1986** [elementor/elementor#24850] [Developers Community]
### Start a discussion

Hello,
I have dynamic URLs like "mysite.com/iam-dynamic/" which do not exist in WordPress. I have to hook via "template_redirect" and load the Elementor page with PHP. In my function I use "get_header()", "get_footer()" to load these Template parts. This works however I am facing a problem: The drop down navigation in the header does't work. I figured out, that several important JS-files are not loaded.

Is my approach wrong? How can I load Elementor with PHP the right way with all assets? I use current Elementor Pro (current version), "Hello Elementor"-theme and theme builder for header and footer section. Thank you in advance!
Best regards

**Comment by ghost** [comment]
If I understood your question. Perhaps you can create a widget for Elementor, which will render the custom PHP page you want. That way both Elementor parts (header, footer, ...) and custom PHP code will work.
